test/test_parser.py

```diff
@@ -12,7 +12,7 @@ def prepend_dir(file):
 class TestParsers(unittest.TestCase):
     def setUp(self):
         # Append current path to locate files
-        meshes = pywavefront.Wavefront(prepend_dir('simple.obj'))
+        meshes = pywavefront.Wavefront(prepend_dir('simple.obj'), collect_faces=True, cache=False)
```

Comment:
I would make a separate test class for ``collect_faces``. This way we test the parser with and without the new feature.

```py
class TestParserCollectFaces(TestParsers):
    # Override setup
    def setUp(self):
        meshes = pywavefront.Wavefront(prepend_dir('simple.obj'), collect_faces=True)
        self.mesh1 = meshes.mesh_list[0]
        self.mesh2 = meshes.mesh_list[1]

    # ... add your own test methods here ...
```

Reply:
The reason you are getting errors is that other tests also extend ``TestParsers`` overriding the ``setUp`` methods, so they sabotage they tests you added in ``TestParsers``. That is why you need to make your own class for testing the feature 😄

Reply:
Alright! Will do.